### DISCUSSION TITLE
Blank racks graphical represention after some seconds

### DISCUSSION THREAD

**Discussion by rezib** [rackslab/Slurm-web#327] [Q&A] [answered]
This is the first time of loading, it plots the drawing with the racks (although they are not listed in the table below): 

![image](https://github.com/user-attachments/assets/51328d66-532a-4285-85dc-786a0b086b3d)

Then after some seconds the JS function is pulling again the nodes and refreshes the page with a similar error and goes blank: 

![image](https://github.com/user-attachments/assets/6a01acbb-c5bc-4984-b083-2787baf64922)

The content of the page after that is: 

```
<!--
  Copyright (c) 2023-2024 Rackslab

  This file is part of Slurm-web.

  SPDX-License-Identifier: GPL-3.0-or-later
-->
<!DOCTYPE html>
<html class="h-full bg-white">
  <head>
    <meta charset="UTF-8">
    <link rel="icon" href="[/favicon.ico](http://10.56.35.121:5011/favicon.ico)">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Slurm-web</title>
    <script type="module" crossorigin src="[/assets/index-45f37282.js](http://10.56.35.121:5011/assets/index-45f37282.js)"></script>
    <link rel="stylesheet" href="[/assets/index-db70270e.css](http://10.56.35.121:5011/assets/index-db70270e.css)">
  </head>
  <body class="h-full">
    <div id="app"></div>
    
  </body>
</html>
```

_Originally posted by @parapar in https://github.com/rackslab/Slurm-web/discussions/324#discussioncomment-10198742_

**Comment by rezib** [comment]
@parapar What is the version of Slurm you are using on this cluster?

I also suspect unsupported nodes naming scheme. What are the names of your nodes (or similar example if confidentiality matters for you)?

**Comment by parapar** [comment]
Slurm version: 24.05.1
Node names: 
```
sinfo
PARTITION AVAIL  TIMELIMIT  NODES  STATE NODELIST
allnodes*    up   infinite      7  down* arod,c5,gondor,melkor,mordor,shelob,urukhai
allnodes*    up   infinite     22   idle aragorn,aule,c[1-4],elrond,elros,eru,galadriel,gandalf,irmo,manwe,middle,namo,olorin,orome,saruman,sauron,tulkas,ulmo,vana
jdk21        up   infinite      2  down* melkor,mordor
jdk21        up   infinite     17   idle aragorn,aule,c[2-4],elrond,elros,eru,irmo,manwe,namo,orome,saruman,sauron,tulkas,ulmo,vana
students     up   infinite      1   idle middle
```

**Reply by rezib** [reply, marked as answer]
OK, thank you for the reply. I can reproduce the issue in my development environment and in CI. The bug is actually due to Slurm-web not supporting node names without digit when folding/expanding nodesets. I just opened issue #328 to fix the bug in next release.
